### PR TITLE
Add phone number

### LINKED ISSUE #49: Tambahkan no. HP
tambahkan info nomor handphone (no. HP) pengguna. Bersama dengan email, admin menggunakan no. HP untuk menghubungi peminjam bilamana buku telat atau tidak dikembalikan.

- [x] tambah input baru untuk memasukkan no. HP di form signup
- [ ] tambahkan link wa.me yang mengarah ke no. WA pengguna 
- ~[ ] tambahkan provider verifikasi no. HP di Supabase~
- ~[ ] implementasi verifikasi no. HP~

### PR DESCRIPTION
**Pull request by Adreon19**
Screenshot 2023-04-02 225057
i miss her

### CODE REVIEW

**Comment by Klrfl** [comment]
What

**Comment by Adreon19** [comment]
yes

**Comment by Adreon19** [comment]
grace

**Comment by Klrfl** [comment]
Eh input no HPnya kasih limit dong, 14 aja maks panjangnya, terus gak pakai koma bisa gak

**Comment by Adreon19** [comment]
ay ay

**Comment by Adreon19** [comment]
https://github.com/user-attachments/assets/909b4cd0-e2b1-4ed0-9418-05470be3f48e

**Comment by Adreon19** [comment]
https://github.com/user-attachments/assets/0825ff4c-d6f9-42b7-a7a2-ea755082b494

File: components/signIn/SignInForm.vue [hunk 1]
```diff
@@ -147,6 +147,7 @@ async function handleSignUp() {
         <label for="signup-number">No. HP</label>
         <InputText
           id="signup-number"
+          v-model="data.phoneNumber"
```
Comment (Klrfl) [inline components/signIn/SignInForm.vue:150]: Bruh

**Review by Klrfl — APPROVED** [review]
Commit is off topic but its ok

**Pull request merged**